src/Resources/Resources/ChangeLog.md

```diff
@@ -24,6 +24,7 @@
 * Introduced secrets detection feature to safeguard sensitive data.
 * Migrated SDK generation from autorest csharp to autorest powershell.
 * Fixed deployment and deployment stack New/Set cmdlets to fail if template/parameter uri fails to downloads.
+* Adjusted Depoyment Stacks cmdlets to prepare for GA.
 
```

Comment:
Do you have any reasons of adding changelogs under those 2 versions instead of under Upcoming Release?